pkg/api/mlflow/common/helpers_test.go

```diff
@@ -0,0 +1,42 @@
+package common
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// TestGetContentType the TestGetContentType private helpers
+func TestGetContentType(t *testing.T) {
+	testCases := []struct {
+		name     string
+		filename string
+		expected string
+	}{
+		{
+			name:     "FromListOfTextTypes",
```

Comment:
it will be of course nice to text all the extensions if it possible.

Reply:
sample that shows code is working as expected is correct, yes?